src/components/EmptyState/Content.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import * as Rivet from "../util/Rivet";
+
+const Content = ({ children, ...attrs }) => (
+  <div {...attrs} className="rvt-empty-state__content">
```

Comment:
Should `className` merge (rather than replace) classes brought in with `attrs.className`? Is this the pattern done with other components?